user_manual/changes/UC02-047.yaml

```diff
@@ -0,0 +1,5 @@
+type: bugfix
+title: Fix `EntrySpec` xref_equation.
+description: |
+    This change fixes the xref_equation for `EntrySpec` name resolution.
```

Comment:
These change entries do not target LAL developers but LAL users, and they have no clue what you are talking about here :D

Reply:
Should I remove it or move it somewhere else?

Reply:
No you should rewrite it with a different wording, such that users can understand it (they don't know what xref_equation is). From their POV, what didn't work before but will work now thanks to your patch?

Reply:
Sure, sorry about that.

Reply:
Hope it's better now?